antarest/matrixstore/service.py

```diff
@@ -469,3 +474,14 @@ def download_matrix(
         matrix = self.get(matrix_id)
         if matrix:
             write_tsv_matrix_to_file(matrix, str(tmp_file))
+
+    def update_matrix(
+        self,
+        matrix_data: List[List[MatrixData]],
+        slices: List[MatrixSlice],
+        operation: Operation,
+    ) -> str:
+        new_matrix_data = MatrixEditor.update_matrix_content_with_slices(
+            matrix_data=matrix_data, slices=slices, operation=operation
+        )
+        return self.create(new_matrix_data)
```

Comment:
or you could put that in the MatrixManagement class, or in the interface/abstract class ISimpleMatrixService